src/components/Steps/index.js

```diff
@@ -44,57 +67,78 @@ class Steps extends React.Component {
+                                ref={this.state.steps[index]}
+                            >
+                                {prevStep && (
+                                    <div
+                                        className="navigator"
+                                        onClick={() => this.scrollTo(prevStep)}
+                                    >
+                                        <ScrollMouseIcon />
+                                        {/* <i
+                                         className="arrow fa fa-2x fa-arrow-up"
+                                         data-tip={prevStep.name}
+                                         /> */}
+                                    </div>
+                                )}
+                                <div className="step-header">
+                                    {name}
                                 </div>
-                            )}
-                            <div className="step-header">
-                                {name}
-                            </div>
-                            <div className="step-content">
-                                <StepComponent
-                                    scrollToPrev={() => this.scrollTo(prevStep)}
-                                    scrollToNext={() => this.scrollTo(nextStep)}
-                                />
-                            </div>
-                            {nextStep && (
-                                <div
-                                    className="navigator"
-                                    onClick={() => this.scrollTo(nextStep)}
-                                >
-                                    {/* <i
-                                        data-tip={nextStep.name}
-                                        className="arrow fa fa-2x fa-arrow-down"
-                                    /> */}
-                                    <ScrollMouseIcon />
+                                <div className="step-content">
+                                    <StepComponent
+                                        scrollToPrev={() => this.scrollTo(prevStep)}
+                                        scrollToNext={() => this.scrollTo(nextStep)}
+                                    />
+                                    { name === 'Choose additional criteria' ? <span className="map-button" onClick={this.displayMap}>SHOW MAP</span> : null}
                                 </div>
-                            )}
-                        </div>
-                    );
+                                {nextStep && (
+                                    <div
+                                        className="navigator"
+                                        onClick={() => this.scrollTo(nextStep)}
+                                    >
+                                        {/* <i
+                                         data-tip={nextStep.name}
```

Comment:
Please remove this comment, I forgot to remove it.

Reply:
XD